content/stylesheets/components/_avatar.scss

```diff
@@ -17,6 +17,10 @@
     height: 100%;
     object-fit: cover;
     object-position: center;
+
+    &[src=""] {
```

Comment:
it's better to not have such images in markup, @timja already fixed the .rb file, I'd suggest removing this rule from CSS

Reply:
Reverted 👍 Cheers both